my-app/src/Components/cartItem/index.jsx

```diff
@@ -1,15 +1,35 @@
-import * as React from 'react';
+import * as React from "react";
 
 export class CartItem extends React.Component {
-    render() {
-        return(
-            <div className="cart-item">
-                <span>{this.props.peak} |</span>    
-                <span> {this.props.county} |</span>    
-                <span> {this.props.cottage}</span>    
-            </div>
-        );
-    }
-        
-
-}
\ No newline at end of file
+  shouldComponentUpdate(nextProps) {
```

Comment:
stiu ca asa v-am aratat exemplul, e ok. Dar daca verifici aici toate proprietatiile de pe props, poti efectiv sa convertesti componenta in Pure, adica sa ai export class CartItem extends React.PureComponent. (am amintit de ea pe final)
Despre PureComponent: https://reactjs.org/docs/react-api.html#reactpurecomponent